Make.defs

```diff
@@ -152,3 +152,11 @@ endif
 
 CFLAGS   += ${shell $(INCDIR) $(INCDIROPT) "$(CC)" "$(APPDIR)$(DELIM)include"}
 CXXFLAGS += ${shell $(INCDIR) $(INCDIROPT) "$(CC)" "$(APPDIR)$(DELIM)include"}
+
+ifneq ($(CONFIG_BUILD_KERNEL),y)
```

Comment:
Need remove ifneq ($(CONFIG_BUILD_KERNEL),y), libapps.a is also needed for kernel build.

Reply:
done